lib/hammer_cli_foreman_ansible.rb

```diff
@@ -14,6 +15,8 @@ module HammerCLIForemanAnsible
 
   require 'hammer_cli_foreman_ansible/command_extensions'
 
+  HammerCLIForemanRemoteExecution::JobTemplate::InfoCommand.extend_with(HammerCLIForemanAnsible::CommandExtensions::JobTemplate.new)
```

Comment:
@adamruzicka 
I'm not sure if that's the best place to put it.
Should I put it instead in a new file: `lib/hammer_cli_foreman_ansible/job_template.rb`?

Reply:
I'd say either here or at the end of `lib/hammer_cli_foreman_ansible/command_extensions.rb` is fine. Having a file with a single line is probably an overkill

Reply:
Then I'll leave it here. Thanks!